comb_spec_searcher/rule_db/base.py

```diff
@@ -345,7 +361,15 @@ class RuleDB(RuleDBBase):
     def __init__(self) -> None:
         super().__init__()
         self._rule_to_strategy: Dict[Tuple[int, Tuple[int, ...]], AbstractStrategy] = {}
+        self._eqv_rule_to_strategy: Dict[Tuple[int, Tuple[int, ...]], AbstractStrategy]
+        self._eqv_rule_to_strategy = {}
```

Comment:
this can be one line

Reply:
Yes but the then black formatting make it ugly :smirk: